src/compute/tests/cdc_tests.rs

```diff
@@ -131,44 +122,37 @@ impl MockOffsetGenExecutor {
     }
 }
 
-impl Executor for MockOffsetGenExecutor {
+impl Execute for MockOffsetGenExecutor {
     fn execute(self: Box<Self>) -> BoxedMessageStream {
         self.execute_inner().boxed()
     }
-
-    fn schema(&self) -> &Schema {
-        &self.schema
-    }
-
-    fn pk_indices(&self) -> PkIndicesRef<'_> {
-        &self.pk_indices
-    }
-
-    fn identity(&self) -> &str {
-        &self.identity
-    }
 }
 
 #[tokio::test]
 async fn test_cdc_backfill() -> StreamResult<()> {
```

Comment:
@StrikeW Please help check if this test is as expected after change, thanks!

Reply:
confirmed. I will add assert in the test later.